src/components/common.js

```diff
@@ -31,7 +31,7 @@ export const Clickable = ({ as = 'div', disabled, tooltip, tooltipSide, onClick,
 }
 
 export const link = function(props, children) {
-  return h(Interactive,
+  return h(Clickable,
```

Comment:
I'm not sure about this. Clickable deals with `onClick` handlers, which aren't relevant to most links.